FiveCalls/FiveCalls/AboutViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AboutViewController.swift
+//  FiveCalls
+//
+//  Created by Alex on 2/6/17.
+//  Copyright © 2017 5calls. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AboutViewController : UIViewController, UIWebViewDelegate {
+    
+    @IBOutlet weak var contentView: UIWebView!
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+
+    @IBAction func close(_ sender: UIBarButtonItem) {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    override func viewDidLoad() {
+        if let asset = NSDataAsset(name: "AboutHtml") {
+            let htmlString = String(data: asset.data, encoding: .utf8)!
+            print(htmlString)
+            contentView.loadHTMLString(htmlString, baseURL: nil)
+            contentView.delegate = self
+        }
+    }
+    
+    func webView(_ webView: UIWebView, shouldStartLoadWith request: URLRequest, navigationType: UIWebViewNavigationType) -> Bool {
+        switch navigationType {
+        case .linkClicked:
+            // Open links in Safari
+            guard let url = request.url else { return true }
+            
+            if #available(iOS 10.0, *) {
+                UIApplication.shared.open(url, options: [:], completionHandler: nil)
+            } else {
+                // openURL(_:) is deprecated in iOS 10+.
```

Comment:
Oh forgot about this... my calling of `openURL:completion` call of this earlier is iOS 10 only. I'll have to fix that.